src/scss/components/Workbench.scss

```diff
@@ -1,87 +1,122 @@
+.workbenchView {
+  display: grid;
+  grid-template-columns: auto min-content;
+  grid-template-rows: minmax(41px, min-content) auto;
+  grid-template-areas:
+    "parameters view-actions"
+    "inner inner";
+  overflow: hidden;
+  position: relative;
+  gap: 5px;
+  margin: .5rem;
+  padding: 5px;
+  border: 1px solid lightgray;
 
-  .workbenchView {
+  & > .view-actions {
+    grid-area: view-actions;
     display: flex;
-    width: 100%;
-    overflow-y: auto;
-    overflow-x: auto;
-    margin: 1.5rem;
-    border: 1px solid lightgray;
-  }
-
-  .workbenchWrapper {
-    flex: 1 1 auto;
+    justify-content: flex-end;
+    justify-self: flex-end;
   }
-  
-  .Pane {
-    display: flex;
-    min-height: 300px;
-    min-width: 300px;
+  & > .view-parameters {
+    margin-bottom: 0;
+    grid-area: parameters;
+    opacity: 1;
+    transition: opacity 0.3s ease;
   }
 
-  .Pane2 {
-    flex: 0 1 auto !important;
+  & > .inner {
+    grid-area: inner;
+    position: relative;
+    border-top: 1px solid #ddd;
+    padding-top: 5px;
+    overflow-y: auto;
+    height: 0;
+    min-height: 100%;
+    display: grid;
+    grid-template-columns: minmax(0, 1fr);
   }
+  @import '~tdp_core/dist/scss/components/view';
+}
 
-  .SplitPane {
-    width: 100% !important;
-  }
+.workbenchWrapper {
+  flex: 1 1 auto;
+}
 
-  .gutter {
-    background-color: #eee;
-    background-repeat: no-repeat;
-    background-position: 50%;
-  }
+.view-title {
+  font-size: 16px;
+}
 
-  .workbenchOverlayButton {
-    opacity: 1;
-  }
+.Pane {
```

Comment:
Why do some CSS classes start with a upper-case letter? Is this a requirement from a dependent library? Otherwise please rename these CSS class names.

Reply:
Yes, this is me overriding classes that come from the react-split library